packages/charting/src/components/VerticalStackedBarChart/examples/VerticalStackedBarChart.Styled.Example.tsx

```diff
@@ -76,15 +76,15 @@ export class VerticalStackedBarChartStyledExample extends React.Component<{}, {}
             allowFocusOnLegends: true,
             styles: {
               rect: {
-                borderRadius: "3px"
-              }
-            }
+                borderRadius: '3px',
```

Comment:
Let's revert these changes, they are not relevant anyhow to this PR.

Reply:
somehow commit always change formatting this file... I don't know how to fix it.

Reply:
with merge latest master it get fixed